coderd/templateversions.go

```diff
@@ -1463,6 +1512,27 @@ func (api *API) postTemplateVersionsByOrganization(rw http.ResponseWriter, r *ht
 			return err
 		}
 
+		// Check for eligible provisioners. This allows us to log a message warning deployment administrators
+		// of users submitting jobs for which no provisioners are available.
+		allProvisioners, activeProvisioners, err := checkProvisioners(ctx, tx, organization.ID, tags, api.DeploymentValues.Provisioner.DaemonPollInterval.Value())
```

Comment:
from sync with Sas: return a TemplateVersionWarning again as this avoids clients having to duplicate this check